### ISSUE TITLE
Is supporting empty lines possible in Anki?

### ISSUE THREAD

**Issue by kodezen** [robbielaldrich/ankiTypebox#6]
Had a quick question about supporting empty lines and was wondering if you experimented with it while developing the addon. 

My use case: I've been using typebox to practice small code snippets. I noticed that typebox doesn't recognize solutions with empty lines as correct. Meaning that I can't create a solution like this: 

```
statement a
[empty line]
statement b
```

Answer is marked as incorrect if you try to answer the card in the same manner. Cards formatted without empty lines work as expected
```
statement a
statement b
```

I've been accounting for this by not spacing out function definitions, ex:

```
def a():
    pass
def b():
    pass
```

But was wondering if it's somehow possible to support the correct matching of cards with empty lines? Is this even doable? Or is it an Anki limitation?

**Comment by robbielaldrich** [comment]
Hey, I just made an update that should respect the newlines in your answer when it does the correcting. Please try updating and let me know if you run into other issues!

**Comment by kodezen** [comment]
Hey this is great thank you so much! Just tried it out and it works, so cool! I did run into one issue with looks when it comes to copy pasting multi-line solutions (unrelated to blank lines) from other sources like a text editor/IDE. Unfortunately it seems like all my multi-line cards that have been created this way are now broken, single-line cards are unaffected and work as expected

Made a loop to walk you through it / explain what's going on cause I figured it would be a little tough to explain through text! Let me know if this helps or if there's any other info I can provide!

https://www.loom.com/share/061742f611e24ee6995e551ac8ed05f8

**Comment by robbielaldrich** [comment]
I found that text pasted into notes is represented by Anki in different HTML than if the text is entered manually. Just made an update to handle this. There might still be other issues when the text is pasted or added in different ways... Thanks for your help for now though!

**Comment by kodezen** [comment]
Of course I'm glad to help! I just tried it out and unfortunately still getting the issue but I think the newly added code will work with a very tiny change. I wrote the value of _cors_ to a file and the value was:

x<br>y

For a card with pasted field:
x
y

Looks like I have <br> tags but no <pre> and </pre> for the pasted multi-line field on the card, meaning that the <br> --> \n substitution isn't performed (if block not entered). Hope this helps 😃
